change event on input

MerlinMason opened the pull request:
Issue #23

MerlinMason committed:
change event on input #23

jenwilhelm commented:
@MerlinBB Hello, thanks for submitting this pull request. I just took a peek and it looks like this is only being triggered if a user focuses on the input element first. If I load the page and click immediately on the picker to insert an emoji, the event is not fired.

I think moving this to the `emojiClicked` function on line 205 should do the trick. :cactus:

MerlinMason committed:
keyup event trigger and notes added to demo

MerlinMason commented:
Hey Jen,

ahh, sorry, didn't spot that... changed to as you suggested and seems to be working now.

Also after some thought I was thinking that "keyup" would be a more intuitive event to trigger as this is most often what people bind to, thoughts?

Furthermore, added a quick note / function in the demo.html file to console.log() the input val().

jenwilhelm commented:
@MerlinBB Awesome, works like a charm. `keyup` is probably the better event to bind to, good thinking. :+1: 

Merging in now! :pineapple:
